### PR TITLE
Portfolio middle section

### PR DESCRIPTION
**Pull request by souad988**
Create works section using grid layout and flexbox
Create about me section 
Add about myself categories(skills, framework ...) using unordered lists
Implement the button interactions

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -44,5 +44,154 @@ <h2 class="headline-2">I’m a software developer</h2>
...
+                        <img src="../media/images/aboutme_languages.png" alt="languages icon"/>
+                    </div>
+                    <div class="languages_section">
+                        <p>Languages</p>
+                        <div class="languages_list">
+                            <div class="language"><p>JavaScript</p></div>
+                            <div class="language"><p>Ruby</p></div>
+                            <div class="language"><p>html</p></div>
+                            <div class="language"><p>css</p></div>
+                        </div>
```
Comment (Whoistolu) [inline src/index.html:155]: - [ ] [REQUIRED] One of the requirements says your list of technologies should be embedded in a `ul` tag and each language in a `li`tag. This makes your code more semantic.

File: src/index.html [hunk 2]
```diff
@@ -44,5 +44,154 @@ <h2 class="headline-2">I’m a software developer</h2>
...
+                    </div>
+                    <div class="languages_section">
+                        <p>Frameworks</p>
+                        <div class="languages_list">
+                            <div class="language"><p>Bootstrap</p></div>
+                            <div class="language"><p>Ruby on rails</p></div>
+                            <div class="language"><p>RSpec</p></div>
+                            <div class="language"><p>Capybara</p></div>
+                            <div class="language"><p>Selenium</p></div>
+                        </div>
```
Comment (Whoistolu) [inline src/index.html:172]: - [ ] [REQUIRED] Kindly refer to my comment concerning the language list been embedded in a `ul` and `li` tags.

File: src/index.html [hunk 3]
```diff
@@ -44,5 +44,154 @@ <h2 class="headline-2">I’m a software developer</h2>
...
+                    </div>
+                    <div class="languages_section">
+                        <p>Skills</p>
+                        <div class="languages_list">
+                            <div class="language"><p>Codekit</p></div>
+                            <div class="language"><p>GitHub</p></div>
+                            <div class="language"><p>Codepen</p></div>
+                            <div class="language"><p>Gitlab</p></div>
+                            <div class="language"><p>Terminal</p></div>
+                        </div>
```
Comment (Whoistolu) [inline src/index.html:189]: - [ ] [REQUIRED] Kindly refer to my comment concerning the language list been embedded in a `ul` and `li` tags.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @souad988 ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] [REQUIRED] Kindly add a descriptive README to your project. This will make you more professional and it is also a file that gives a user knowledge of what your project entails.
Read [this](https://github.com/microverseinc/readme-template) to see what the Mivroverse descriptive README is.

- [ ] [OPTIONAL] It will be nice to start the first letter of your commit messages with a capital letter. This is in accordance with professionalism.

- [ ] [REQUIRED] Kindly ensure you stick to the figma ma template design as much as possible. Please fix the link to the social media icons on your design as they seem to be broken
Also, reduce the sizes of the background images in your About me sections so that it looks like the figma design and remove the outline on your See Project and Get my resume `buttons` so it also looks like the ones in the figma template. See below:
### Your design
your-design1

your-design2
### Figma
figma2




_Check the inline comments under the review for sections to improve._

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/index.html [hunk 4]
```diff
@@ -35,14 +35,163 @@ <h2 class="headline-2">I’m a software developer</h2>
...
+                        <img src="../media/images/aboutme_languages.png" alt="languages icon"/>
+                    </div>
+                    <div class="languages_section">
+                        <p>Languages</p>
+                        <ul class="languages_list">
+                            <li class="language"><p>JavaScript</p></li>
+                            <li class="language"><p>Ruby</p></li>
+                            <li class="language"><p>html</p></li>
+                            <li class="language"><p>css</p></li>
+                            </ul>
```
Comment (mateo951) [inline src/index.html:155]: Great! Your changes meet with the previous reviewer requirement 🔥😎

File: src/index.html (hunk 1 above)
Comment (mateo951) [reply src/index.html]: Addressed 👌

File: src/index.html (hunk 2 above)
Comment (mateo951) [reply src/index.html]: Addressed 🔥😎

File: src/index.html (hunk 3 above)
Comment (mateo951) [reply src/index.html]: Addressed 🙌

**Comment by mateo951** [comment]
Please, as the previous reviewer mentioned, it's important that you add a descriptive README to your project.

**Review by mateo951 — CHANGES_REQUESTED** [review]
Hi @souad988, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there is one more issue that isn't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mateo951** [comment]
Please, as the previous reviewer mentioned, it's important that you add a descriptive README to your project.

**Comment by souad988** [comment]
> 

i would like to have more info, i changed my readme file couple time, would you specify please ?

**Comment by mateo951** [comment]
Just noticed that your proper README file is on the main branch! I was looking at the feature branch 👀 Sorry for that mistake @souad988! Your README is actually awesome. Please ask for another review so they can approve your project

**Pull request merged**